kafkamgmt/apiv1/client/docs/KafkaRequest.md

```diff
@@ -19,13 +19,14 @@ Name | Type | Description | Notes
 **FailedReason** | Pointer to **string** |  | [optional] 
 **Version** | Pointer to **string** |  | [optional] 
 **InstanceType** | Pointer to **string** |  | [optional] 
+**ReauthenticationEnabled** | **bool** |  | 
 
 
 ## Methods
 
 ### NewKafkaRequest
 
-`func NewKafkaRequest() *KafkaRequest`
+`func NewKafkaRequest(multiAz bool, reauthenticationEnabled bool, ) *KafkaRequest`
```

Comment:
cc @rkpattnaik780 @wtrocki a breaking change we will probably need to adjust to in the CLI.